README.md

```diff
@@ -1,7 +1,8 @@
 remodel
 =======
 
-[![wercker status](https://app.wercker.com/status/572156625000100a804563e52fbe3dfd/s/master "wercker status")](https://app.wercker.com/project/bykey/572156625000100a804563e52fbe3dfd)
+[![wercker status](https://app.wercker.com/status/6bb7e818f8c1efd7ca9bc4bce91a59de/m "wercker status")](https://app.wercker.com/project/bykey/6bb7e818f8c1efd7ca9bc4bce91a59de)
```

Comment:
Inadvertent. It was my first time using wercker and had just changed it for my fork while exploring the tool.